source/transfer_list.rst

```diff
@@ -993,6 +995,48 @@ into its memory map during platform setup. If other memory types are required
      - hdr_size + 0x8
      - The size of the memory region.
 
+**FF-A SP binary (XFERLIST_FFA_SP_BINARY)**
+
+This entry holds a reference to an FF-A secure partition's binary.
+
+Hafnium SPMC supports the SP's artefacts to be packaged as a Transfer List
+generated at build time (with TF-A's tlc tool).
+
+The respective Transfer List contents: SP binary/image, the FF-A manifest, and
+other resoures (e.g. HOB - optional).
+
+This entry type helps identify the executable that SPMC should bootstrap/hand-over
```

Comment:
Could you provide more context or clarify what you're trying to say? It would be helpful to explain how this entry type is intended to be used, for the benefit of others who may be unsure.